Formula/gauge.rb

```diff
@@ -1,6 +1,6 @@
 class Gauge < Formula
   desc "Test automation tool that supports executable documentation"
-  homepage "https://getgauge.io"
+  homepage "https://gauge.org"
```

Comment:
The gauge homepage seems to have changed to https://gauge.org, so I went ahead and updated the homepage as well in this commit.

This is the only change in this PR that isn't separating shell command arguments, so I figured I'd point it out so it isn't missed.